examples/step-33/step-33.cc

```diff
@@ -1765,37 +1772,54 @@ namespace Step33
     typedef Sacado::Fad::DFad<double> FluxMatrix[EulerEquations<dim>::n_components][dim];
     FluxMatrix *flux = new FluxMatrix[n_q_points];
 
+    typedef double FluxMatrixOld[EulerEquations<dim>::n_components][dim];
+    FluxMatrixOld  *flux_old = new FluxMatrixOld[n_q_points];
+
     typedef Sacado::Fad::DFad<double> ForcingVector[EulerEquations<dim>::n_components];
     ForcingVector *forcing = new ForcingVector[n_q_points];
 
+    typedef double ForcingVectorOld[EulerEquations<dim>::n_components];
+    ForcingVectorOld *forcing_old = new ForcingVectorOld[n_q_points];
+
```

Comment:
I know you just followed the old programming style, but I would prefer if these variables (flux, flux_old, etc.) would not handle memory allocation on their own but rather use std::vector<FluxMatrix> types.